app/config/collections.php

```diff
@@ -2764,6 +2764,33 @@
             ],
         ]
     ],
+    'cache' => [
+        '$collection' => Database::METADATA,
+        '$id' => 'cache',
+        'name' => 'Cache',
+        'attributes' => [
+            [
+                '$id' => 'accessedAt',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => false,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+         ],
+        'indexes' => [
+            [
+                '$id' => '_key_accessed',
```

Comment:
Is this key name matching our other conventions?

Reply:
changed